users/models.py

```diff
@@ -40,6 +52,10 @@ class User(AbstractBaseUser, PermissionsMixin, TimeStamp):
     USERNAME_FIELD = "email"
     REQUIRED_FIELDS = ["name"]
 
+    # pylint: disable=no-member
+    role = models.ForeignKey(Role, on_delete=models.CASCADE, default=1)
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = ["name"]
```

Comment:
Weren't these fields added in the User management feature PR?

Reply:
Yes, they were. I think after adding the `Role` model git thinks these lines are additions